src/utils/validation.ts

```diff
@@ -122,6 +122,13 @@ export async function validateResourcesFromFolder(resources: Resource[], root: F
     return null;
   }
 
+  const resourcesRelative = resources.map(resource => {
```

Comment:
Not sure how often this function will run. No need to look into this now as we have time constraints, but this feels more like a parsing concern? As in, the files we parse to extractK8sResources should have their file paths updated.